core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/machine/Server.java

```diff
@@ -25,25 +25,35 @@
     String getRef();
 
     /**
-     * Address of the server in form <b>host:port</b>
+     * External address of the server in form <b>host:port</b>
```

Comment:
Line break by entering 'enter' key doesn't work in javadocs. Please add dot and `<p/>` or `<br/>`